bundles/binding/org.openhab.binding.fritzboxtr064/.classpath

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" output="target/classes" path="src/main/java"/>
+	<classpathentry excluding="**" kind="src" output="target/classes" path="src/main/resources"/>
+	<classpathentry exported="true" kind="lib" path="lib/commons-codec-1.9.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/commons-logging-1.2.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/fluent-hc-4.4.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/httpclient-cache-4.4.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/httpclient-win-4.4.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/httpmime-4.4.1.jar"/>
```

Comment:
Dont understand how to fix this. If I delete the dependency it cannot be compiled without it (of course). I searched the project tree for 4.1.3 references but could not find the lib/jar file to reference to.
If I need to add the lib file as I did before, why should I use an older version 4.1.3 instead of newer 4.4.1?